GovUkDesignSystem/GovUkDesignSystemComponents/SummaryList.cshtml

```diff
@@ -2,10 +2,12 @@
 
 <dl class="govuk-summary-list @Model?.Classes" @(Html.Raw(Model.Attributes != null ? string.Join(" ", Model.Attributes.Select(a => a.Key + "=\"" + a.Value + "\"")) : ""))>
     @{
+        var anyRowHasActions = !Model.Rows.All(r => r.Actions?.Items == null || r.Actions.Items.Count == 0);
```

Comment:
Tiny niggle :) using `Model.Rows.Any()` would mean you didn't need the initial `!` and would match the varible name

Reply:
Oh yeah! I was worried about the ? operator on Actions but it will still work, will fix